src/commands/service/pipeline.ts

```diff
@@ -33,6 +33,9 @@ import {
   validateOrgNameThrowable,
   validateProjectNameThrowable,
 } from "../../lib/validator";
+import { validateRepoUrl } from "../project/pipeline";
```

Comment:
If we're sharing this method, we should put it in one of our shared util files instead.

Reply:
I've added this to the gitutils.ts